### PR TITLE
Mat 1462 sellVoucher reverts after slashing

### PR DESCRIPTION
**Pull request by imyourm8**

### CODE REVIEW

File: contracts/staking/validatorShare/ValidatorShare.sol [hunk 1]
```diff
@@ -147,6 +147,10 @@ contract ValidatorShare is IValidatorShare {
             _amount = _amount.sub(_rewards);
         }
 
+        if (_amount > activeAmount) {
+            _amount = activeAmount;
```
Comment (0xAshish) [inline contracts/staking/validatorShare/ValidatorShare.sol:151]: This is wrong.

Comment (imyourm8) [reply contracts/staking/validatorShare/ValidatorShare.sol:151]: Can you elaborate what is wrong and how would you do this?

Comment (0xAshish) [reply contracts/staking/validatorShare/ValidatorShare.sol:151]: bob stakes 200 and alice stakes 100, after which total pool 300 got slashed by 50% then we don't want to give bob 150 instead it should be 100(bob) and 50(Alice).

**Comment by 0xAshish** [comment]
Closing the PR since there are different set of changes required.

**Comment by 0xAshish** [comment]
Closing the PR since there are different sets of changes required.

**Pull request closed without merging**